docs/_includes/ds6/breadcrumb.html

```diff
@@ -0,0 +1,61 @@
+<div id="breadcrumb">
+
+    <h2><span class="secondary-text">@ebay/skin/</span>breadcrumb</h2>
+
+    <h3>Breadcrumb - Last item as the parent</h3>
+    <p>The last item in the breadcrumb list is the parent of the current page. Use this pattern if the current page category is not specific</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation">
+                <h2 id="breadcrumbs_heading" class="clipped">You are here</h2>
+                <ul>
+                    <li><a href="https://www.ebay.com/">ebay</a></li>
+                    <li><a href="https://www.ebay.com/rpp/cell-phone-pda">Cell Phones, Smart Watches & Accessories</a></li>
+                    <li><a href="https://www.ebay.com/b/Smart-Watch-Accessories/182064/bn_16565905">Smart Watch Accessories</a></li>
+                    <li><a href="https://www.ebay.com/b/Smart-Watch-Bands/182068/bn_16565906">Smart Watch Bands</a></li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+    {% highlight html %}
+    <nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation">
```

Comment:
This block should have no leading whitespace. Therefore, this
```
        {% highlight html %}
        <nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation">
```
... should become ...
```
        {% highlight html %}
<nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation">
```
... with all the subsequent code in the highlight blocks adjusting accordingly.

Please follow the example here: https://github.com/eBay/skin/blob/master/docs/_includes/ds6/listbox.html#L37-L63